src/Workspaces/Core/Portable/WorkspacesResources.resx

```diff
@@ -792,18 +792,36 @@
   <data name="Changing_document_0_is_not_supported" xml:space="preserve">
     <value>Changing document '{0}' is not supported.</value>
   </data>
-  <data name="ValueUsageInfo_NameOnly" xml:space="preserve">
-    <value>NameOnly</value>
+  <data name="ValueUsageInfo_Name" xml:space="preserve">
+    <value>Name</value>
   </data>
-  <data name="ValueUsageInfo_Read" xml:space="preserve">
+  <data name="ValueUsageInfo_ValueRead" xml:space="preserve">
     <value>Read</value>
   </data>
-  <data name="ValueUsageInfo_Reference" xml:space="preserve">
+  <data name="ValueUsageInfo_ValueReference" xml:space="preserve">
     <value>Reference</value>
   </data>
-  <data name="ValueUsageInfo_Write" xml:space="preserve">
+  <data name="ValueUsageInfo_ValueWrite" xml:space="preserve">
     <value>Write</value>
   </data>
+  <data name="ValueUsageInfo_DottedName" xml:space="preserve">
+    <value>Qualify</value>
+  </data>
+  <data name="ValueUsageInfo_GenericTypeArgument" xml:space="preserve">
+    <value>Type Argument</value>
+  </data>
+  <data name="ValueUsageInfo_BaseTypeOrInterface" xml:space="preserve">
+    <value>Base</value>
+  </data>
+  <data name="ValueUsageInfo_ObjectCreation" xml:space="preserve">
+    <value>new()</value>
```

Comment:
A few issues here:
1. this seems problematic iven that C# is adding a new language feature with this syntax.
2. this is C# syntax.  VB uses 'New'.

Reply:
Sure, will bring this up ad design meeting.